src/name_bazaar/ui/events/public_resolver_events.cljs

```diff
@@ -69,3 +67,52 @@
      {:db (assoc-in db [:public-resolver/reverse-records
                         addr
                         :public-resolver.record/name] name)})))
+
+(reg-event-fx
+ :public-resolver.name/point
+ [interceptors (validate-first-arg (s/keys :req [:ens.record/name
+                                                 :ens.record/address]))]
+ (fn [{:keys [:db]} [form-data]]
+   (let [instance (get-instance db :ens)
+         form-data (assoc form-data
+                          :ens.record/node (namehash (str (:ens.record/name form-data)
+                                                          constants/registrar-root)))]
+     (info [:POINT-NAME name])
+     {:dispatch [:district0x/make-transaction
+                 {:name (gstring/format "Pointing %s to %s"
+                                        (:ens.record/name form-data)
+                                        (:ens.record/address form-data))
+                  :contract-key :public-resolver
+                  :contract-method :set-addr
+                  :form-data (select-keys form-data [:ens.record/node :ens.record/address])
+                  :args-order [:ens.record/node :public-resolver]
+                  ;;:result-href (path-for :route.ens-record/detail form-data)
+                  :form-id (select-keys form-data [:ens.record/node :ens.record/address])
```

Comment:
just `(select-keys form-data [:ens.record/node])` would be fine here, as 1 name can resolve to only 1 address at the time, so "loading" state should be displayed for a particular name, no matter what address is in input.